ocs_ci/ocs/constants.py

```diff
@@ -1331,6 +1331,7 @@
 # bm dict value is based on each worker BM machine of config 40CPU and 256G/184G RAM
 # azure dict value is based on assumption similar to vmware vms min worker config of 12CPU and 64G RAM
 SCALE_WORKER_DICT = {
+    120: {"aws": 3, "vmware": 3, "bm": 2, "azure": 3, "rhv": 3},
```

Comment:
are we adding more workers? or we are using existing worker nodes by labeling them?
In IPI clusters new machineset will be created and and workers will be added based on this dict, with MS consumer cluster default has 3 workers, 3 infra, so are we using those 3 nodes for scale ? and labeling will be done automatic by Automation?

Reply:
My plan is not to add worker nodes to the cluster with the value above. I was just labeling the existing ones.
With MS, I modify the code here to have three worker nodes(for any value): https://github.com/red-hat-storage/ocs-ci/pull/5919/files#diff-5277af9769e0fadc38a711535d8e3996b06f51b54015bf9b8e176b58e7f53ffaR284.